database.go

```diff
@@ -59,6 +59,25 @@ func isKnownArch(arch string) bool {
 	return false
 }
 
+// normalizeOSAndVersion will only include the OSVersion if present.
+// The formart of OS on the platform specifier is <OS>[(<OSVersion>)]
+// OSVersion is optional only and is currently used only by windows OS.
+// If OsVersion is not specified, empty string is returned.
```

Comment:
(Two different ideas about this function description)
Nits and typo-fix only version.
```suggestion
// normalizeOSAndVersion will only return an OSVersion if present.
// The format of OS in the platform specifier is <OS>[(<OSVersion>)]
// OSVersion is optional and is currently used only when OS is "windows".
// If OSVersion is not specified, empty string is returned as the second return value.
```